src/pudl/settings.py

```diff
@@ -401,6 +401,19 @@ class EiaAeoSettings(GenericDatasetSettings):
     years: list[int] = data_source.working_partitions["years"]
 
 
+class VCERARESettings(GenericDatasetSettings):
```

Comment:
I think the way we've named the other sources is with Pascal case (VceRareSettings)